pkg/install.go

```diff
@@ -309,11 +308,18 @@ func SetEngineVar(chaosEngine *v1alpha1.ChaosEngine, testsDetails *types.TestDet
 		chaosEngine.Spec.Experiments[0].Spec.Components.NodeSelector["kubernetes.io/hostname"] = testsDetails.NodeSelectorName
 	}
 
-	// NODE_LABEL for Node-memory-hog and node-cpu-hog
-	if testsDetails.NodeLabel != "" {
+	// update Target Node Details
+	if testsDetails.TargetNodes != "" {
+		log.Infof("[Info] Target Nodes: %v", testsDetails.TargetNodes)
 		envDetails.SetEnv("TARGET_NODES", testsDetails.TargetNodes)
 	}
 
+	// update Node Label Details
+	if testsDetails.NodeLabel != "" {
```

Comment:
There is one more node label in line 324 how is it different?

Reply:
Fixed

Reply:
Fixed